bundles/model/org.eclipse.smarthome.model.lsp/src/main/java/org/eclipse/smarthome/model/lsp/internal/ModelServer.java

```diff
@@ -66,23 +80,25 @@ public void deactivate() {
                 socket.close();
             }
         } catch (IOException e) {
-            logger.error("Error shutting down the Language Server", e);
+            logger.debug("Error shutting down the Language Server", e);
         }
     }
 
-    private void listen() {
+    private void listen(int port) {
```

Comment:
WRT to the changed logging, shouldn't this method return a success indication (`boolean`), so the caller could at least log one error message that the server is not running / listen?

Reply:
As #4722 fixed the superfluous logging in a much better way, I guess my change to the log severities here became obsolete. I'll simply remove that commit.